msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
+
+* Use "Microsoft Teams" on first reference. Subsequent references can be shortened to ”Teams”.
+* Do not abbreviate "Microsoft" (such as "MS" or "MSFT").
+* Avoid unnecessary capitalization and language errors.
+* Follow the [short description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#short-description).
+* Follow the [long description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#long-description).
+* Clearly describe any limitations, conditions, or exceptions to the functionality, features, and deliverables described in the listing and related materials before the customer acquires your offer. The capabilities you declare must relate to the core functions and description of your offer.
+
+### 4.4 Privacy policy
+
+The policy can be specific to your Teams app or an overall policy for all of your services.
+
+* If you use a generic privacy policy template, you must reference "services", "applications", and "platforms" to include your Teams app and your website or service.
+* Must include how you handle user data storage, retention, and deletion. You also must describe the security controls you use for data protection.
+* Must include your contact information.
+* Should not contain broken, beta, or staging URLs.
+
+### 4.5 Terms of use
+
+Your terms of use should be specific and applicable to your offering.
+
+### 4.6 Support links
+
+Your app's support URLs should not require authentication. For example. users should not have to log in to contact you.
+
+### 4.7 Localization
+
+Your app package must include a localization file that includes language translations that display based on the Teams language settings. The file must conform to the [Teams localization schema](~/concepts/build-and-test/apps-localization.md).
+
+## 5.0 Tabs
+
+If your app includes a tab, following these guidelines can increase the likelihood your app will pass submission.
 
 > [!TIP]
-> Make sure your app displays a loading indicator or some form of warning when your app is taking longer than expected to respond.
+> If you want information on creating a high-quality experience, see the [tab design guidelines](~/tabs/design/tabs.md).
+
+### 5.1 Setup
+
+* Tab setup must not dead-end the new user. A way forward message should be provided to complete the workflow.
+* Authentication should happen during tab setup and not after.
+* Tabs must have a help tab with instructions on how to use and configure your app.
+
+### 5.2 Views
 
-### &#9989; Tab content must not have excessive chrome or layered navigation
+* The sign-in screen area use must not use large logos or display an entire webpage.
+* The canvas can be simplified by breaking them into multiple tabs to reduce the complexity.
+* Personal app tabs should not have a duplicate header. Remove the logo from the iframe since the personal app framework already displays the app icon and app name.
```

Comment:
@heath-hamilton ,  Remove personal app from this sentence - 
•	Tabs should not have a duplicate header. Remove the logo from the iframe since the personal app framework already displays the app icon and app name.